geomet/esri.py

```diff
@@ -160,7 +164,8 @@ def _dump_geojson_polygon(data, srid=None):
                 part_item.append(coord)
         if part_item:
             part_list.append(part_item)
-    srid = _extract_geojson_srid(data) or srid
+    if srid is None:
+        srid = _extract_geojson_srid(obj) or srid
```

Comment:
`obj` should be replaced with `data` on this line. This is what's causing the tests to fail.

Reply:
fixed.